starport/cmd/scaffold_flutter.go

```diff
@@ -17,17 +18,14 @@ func NewScaffoldFlutter() *cobra.Command {
 		RunE:  scaffoldFlutterHandler,
 	}
 
-	c.Flags().StringP(flagPath, "p", "./flutter", "path to scaffold content of the Flutter app")
-
 	return c
 }
 
 func scaffoldFlutterHandler(cmd *cobra.Command, args []string) error {
 	s := clispinner.New().SetText("Scaffolding...")
 	defer s.Stop()
 
-	path, _ := cmd.Flags().GetString(flagPath)
-
+	path := filepath.Join(flagGetPath(cmd), "flutter")
```

Comment:
this changes the behavior of the command. in this version, the command always creates the flutter dir. instead, --path should be the exact path for where flutter files will be added into.